packages/styled-ui-docs/pages/scrollbar.mdx

```diff
@@ -68,6 +68,98 @@ Use the `disabled` attribute, the overflow is clipped, and the rest of the conte
 </Scrollbar>
 ```
 
+### The minimal thumb size
+
+Use the `minThumbSize` attribute to set the minimal size (in pixels) for thumbs.
+
+```jsx
+function MinThumbSizeScroll() {
+  const [minThumbSize, setMinThumbSize] = React.useState('32');
+  const [dataRow, setDataRow] = React.useState('8');
+
+  return (
+    <>
+      <Box marginBottom="2x">
+        <Text marginBottom="1x">The minimal thumb size:</Text>
+        <RadioGroup onChange={e => setMinThumbSize(e.target.value)} value={minThumbSize}>
```

Comment:
`setMinThumbSize(Number(e.target.value))`

Reply:
@tinaClin fixed in  [2d67361]